frontend/apps/app/components/SessionDetailPage/components/Output/components/Artifact/hooks/useRealtimeArtifact.ts

```diff
@@ -0,0 +1,121 @@
+'use client'
+
+import { type Artifact, artifactSchema } from '@liam-hq/db-structure'
+import { useCallback, useEffect, useState } from 'react'
+import * as v from 'valibot'
+import { createClient } from '@/libs/db/client'
+
+const artifactDataSchema = v.object({
+  id: v.pipe(v.string(), v.uuid()),
+  design_session_id: v.pipe(v.string(), v.uuid()),
+  artifact: v.unknown(),
```

Comment:
It looked like artifactSchema would be a good choice here, what do you think?

Reply:
You're absolutely right. I'll correct it!

Reply:
I have corrected this one.
https://github.com/liam-hq/liam/pull/2229/commits/a67c644156480ecbee0f4e8e2f68ff60df64effc#diff-7e28d5bb53b49566110cf9add00e335f1ff05564cbdbd7aadf956b7c50b35358